### PR TITLE
Fix for #82, call OnSharing and return a result after ShareAsync if p…

### LINKED ISSUE #82: Android: Share doesn't return and doesn't call OnSharing if postId is null
If you are creating an issue for a BUG please fill out this information. If you are asking a question or requesting a feature you can delete the sections below. 

**Failure to fill out this information will result in this issue being closed.** If you post a full stack trace in a bug it will be closed, please post it to http://gist.github.com and then post the link here.

## Bug Information

On Android, When sharing to Only Me (not sure if it repros with public post), the call to 

var result = await CrossFacebookClient.Current.ShareAsync(shareContent);

never returns and OnSharing is not called.  Looking at the code it's obvious that it doesn't work if PostId is null (which seems to be a common issue).  I have a pull request coming up to fix it.

Version Number of Plugin: 2.2.1
Device Tested On: Huawei Mate 20 Pro
Simulator Tested On:
Version of VS: 16.7.3
Version of Xamarin:
Versions of other things you are using:

### Steps to reproduce the Behavior

Share a post to Only me, OnSharing is never called

### Expected Behavior

OnSharing to be called

### PR DESCRIPTION
**Pull request by NGumby**
…ostId is null

Please take a moment to fill out the following:

Fixes #82  .

Changes Proposed in this pull request:
- return a result and call OnSharing even if postId is null
-
-

### CODE REVIEW

File: src/Plugin.FacebookClient/Plugin.FacebookClient.csproj [hunk 1]
```diff
@@ -21,9 +21,9 @@
 
     <LangVersion>latest</LangVersion>
 
-    <PackageProjectUrl>https://github.com/CrossGeeks/FacebookClientPlugin</PackageProjectUrl>
+    <PackageProjectUrl>https://github.com/CompuSport/FacebookClientPlugin</PackageProjectUrl>
```
Comment (rdelrosario) [inline src/Plugin.FacebookClient/Plugin.FacebookClient.csproj:24]: It seems you changed the url here, can you discard this.

File: src/Plugin.FacebookClient/Plugin.FacebookClient.csproj [hunk 2]
```diff
@@ -35,6 +35,7 @@
     <Copyright>Copyright 2017 CrossGeeks</Copyright>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageId>Plugin.FacebookClient.CompuSport</PackageId>
```
Comment (rdelrosario) [inline src/Plugin.FacebookClient/Plugin.FacebookClient.csproj:38]: It seems you changed the plugin name here, can you discard this.

File: src/Plugin.FacebookClient/Plugin.FacebookClient.csproj [hunk 3]
```diff
@@ -21,9 +21,9 @@
 
     <LangVersion>latest</LangVersion>
 
-    <PackageProjectUrl>https://github.com/CrossGeeks/FacebookClientPlugin</PackageProjectUrl>
+    <PackageProjectUrl>https://github.com/CompuSport/FacebookClientPlugin</PackageProjectUrl>
     <PackageIconUrl>https://github.com/CrossGeeks/FacebookClientPlugin/blob/master/art/icon.png?raw=true</PackageIconUrl>
-    <RepositoryUrl>https://github.com/CrossGeeks/FacebookClientPlugin</RepositoryUrl>
+    <RepositoryUrl>https://github.com/CompuSport/FacebookClientPlugin</RepositoryUrl>
```
Comment (rdelrosario) [inline src/Plugin.FacebookClient/Plugin.FacebookClient.csproj:26]: here as well.

**Comment by rdelrosario** [comment]
Thanks for your contribution. Added just some small feedback.

**Comment by NGumby** [comment]
I removed the offending commit.  Thanks

**Review by rdelrosario — APPROVED** [review]

**Comment by rdelrosario** [comment]
Thanks!

**Pull request merged**